src/components/MarsApp.css

```diff
@@ -23,3 +23,26 @@
 .page-nav-button:hover{
  background-color: cornflowerblue;
 }
+
+.loader-center{
+  padding: 400px 0;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.loader-center img{
+  animation: rotation 6s infinite linear;
+}
+
+@keyframes rotation {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(365deg);
+  }
+}
```

Comment:
We should be storing CSS next to the file which it's relevant to, so in this case the loader css should be in `src/components/shared/loader/Loader.css`